lib/main.coffee

```diff
@@ -101,18 +101,25 @@ module.exports = Hydrogen =
         unless kernel
             kernel = @kernelManager.getRunningKernelFor language
 
+        message = "No running kernel for language `#{language}` found"
+
         if command is 'interrupt-kernel'
+            unless kernel
+                atom.notifications.addError message
+                return
             kernel.interrupt()
 
         else if command is 'restart-kernel'
+            unless kernel
+                atom.notifications.addError message
+                return
             kernelSpec = kernel.kernelSpec
             @kernelManager.destroyRunningKernel kernel
             @clearResultBubbles()
             @kernelManager.startKernel kernelSpec, grammar, =>
                 @setStatusBarElement()
 
         else if command is 'switch-kernel'
-            kernel = @kernelManager.getRunningKernelFor language
```

Comment:
I'm wrong `SignalListView` uses all the arguments though I think this can be simplified in a future PR.